pkg/aggregator/demultiplexer_mock.go

```diff
@@ -74,6 +82,31 @@ func (a *TestAgentDemultiplexer) WaitForSamples(timeout time.Duration) []metrics
 	}
 }
 
+// WaitEventPlatformEvents returns the event platform events samples received by the demultiplexer.
+func (a *TestAgentDemultiplexer) WaitEventPlatformEvents(eventType string, minEvents int, timeout time.Duration) ([]*message.Message, error) {
+	ticker := time.NewTicker(10 * time.Millisecond)
+	timeoutOn := time.Now().Add(timeout)
+	var savedEvents []*message.Message
+	for {
+		select {
+		case <-ticker.C:
+			allEvents := a.aggregator.GetEventPlatformEvents()
```

Comment:
There is currently no api to get the input channel where the events are sent.

It's doable, but that would require exposing to the track type specific input channel of the EP forwarder: https://github.com/DataDog/datadog-agent/blob/0df53f76a1c204266648199499bfad0af60a3c93/pkg/epforwarder/epforwarder.go#L175

Reply:
Let me see if I can do something about it.

Reply:
Just made a working example here: https://github.com/DataDog/datadog-agent/pull/12092/files

The downside is that we are exposing the EPForwarder track specific input channel that is currently private, so not sure it's better way because of that.